src/components/form/passwordPolicy.js

```diff
@@ -0,0 +1,50 @@
+import { minLength, maxLength } from 'vuelidate/lib/validators'
+
+const hasLowerCase = value => /[a-z]/.test(value)
+const hasUpperCase = value => /[A-Z]/.test(value)
+const hasDigit = value => /[0-9]/.test(value)
+const hasSpecialCharacter = value => /[ !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~]/.test(value)
+
+export default value => {
+  const isValid =
+    minLength(8)(value) &&
+    maxLength(20)(value) &&
```

Comment:
@comteharbour je réitère la question à laquelle il n'y a pas eu de réponse. Il n'y a pas de limite de taille de mot de passe dans la spec.

Reply:
ok. Je vire ça.
NB: dans les fixtures, il faut que tu mettes à jour les mots de passe utilisés pour qu'ils correspondent à la politique des mots de passe, sinon les tests ne passent pas. Pour l'instant, j'ai réduit les contraintes des mots de passe, afin que les tests passent.
N'hésite pas à adapter le fichier `common/src/components/form/passwordPolicy.js` qui regroupe toutes ces contraintes, ainsi que les mots de passe utilisés pour tester le composant y relatif.